tests/test_merge.py

```
@@ -0,0 +1,324 @@
+        result = _end_of_day(dt)
+        assert result == datetime(2026, 3, 27, 23, 59, 59)
+
+    def test_preserves_naive(self):
+        dt = datetime(2026, 3, 27, 10, 30)
+        result = _end_of_day(dt)
+        assert result.tzinfo is None
+
+
+# --- _reconcile_events ---
+
+def _make_event(start_hour, end_hour, title="test", action=None, full_event=None):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_make_event`

(ANN202)